config/backstop/backstopConfig.js

```diff
@@ -9,7 +9,8 @@ const urls = {
 const basicScenario = {
   label: 'test', // name of the test
   url: urls.development,
-  referenceUrl: '', // put here reference to github-pages with ready project
+  referenceUrl:
+    'https://mate-academy.github.io/layout_solutions/colored-blocks/',
```

Comment:
why not in gh-pages for this repo?

Reply:
@roman-mate it is a reference to `golden version`. The tool will compare screenshots from this li with screenshots from student's version